### DISCUSSION TITLE
Tailwind doesn't see classes in my files

### DISCUSSION THREAD

**Discussion by workCheckAn** [tailwindlabs/tailwindcss#9825] [Help]
Hi!
I have a problem with Tailwind v3.2.4.
I have installed it with npm install -D tailwindcss. Then I installed tailwind.config.js with npx tailwindcss init.
Versions of Tailwind in package.json (3.2.4) and in a finished file (2.2.19) are different. How can I FIX this? Because Tailwind doesn't see classes in my files. In finished file I have only base classes.
![image](https://user-images.githubusercontent.com/67582144/201536745-fb450b15-4a4c-47b9-8419-932469586659.png)
![image](https://user-images.githubusercontent.com/67582144/201536634-47a1b8b2-73f2-4371-b54a-b03fd4ff73a0.png)
![image](https://user-images.githubusercontent.com/94995463/201537264-01e8cbe1-be2d-4147-a47e-8f6b30164482.png)

**Comment by wongjn** [comment]
It seems like you still have Tailwind v2 judging from your screenshot of the CSS. Your config seems to be for v2 too. Regardless, compiled CSS from v3 will be different from v2. To get utility class rules to compile, ensure you tell Tailwind about source files that are using Tailwind classes by [configuring the `content` key](https://tailwindcss.com/docs/content-configuration).
